fakenet/listeners/ProxyListener.py

```diff
@@ -198,10 +208,14 @@ def handle(self):
 
         except Exception as e:
             self.server.logger.warning('recv() error: %s' % e.message)
+        
+        # Is the pkt ssl encrypted?
+        is_ssl_encrypted = False
```

Comment:
the `UseSSL` config value can be either 'Yes' or 'No'. We need to be consistent here